code/modules/projectiles/guns/rifles.dm

```diff
@@ -1709,3 +1709,62 @@
 		/obj/item/attachable/magnetic_harness,
 		/obj/item/attachable/bayonet,
 	)
+
+/obj/item/weapon/gun/rifle/autogun
+	name = "\improper Agripinaa-Pattern Autogun"
+	desc = "The Autogun is a fully automatic Auto Weapon that fires solid-slug ballistic rounds at the intended target in rapid succession. Cheap, rugged and reliable, it can be found throughout the Imperium of Man, where it is appreciated for its decent damage and accuracy."
```

Comment:
```suggestion
	desc = "The Autogun is a fully automatic firearm that fires solid-slug ballistic rounds at the intended target in rapid succession. Cheap, rugged and reliable, it can be found throughout the Imperium of Man, where it is appreciated for its decent damage and accuracy."
```